### PR TITLE
feat(copy): add word-end, half-page and count motions

### PR DESCRIPTION
**Pull request by yosephbernandus**
Copy mode gains a word-end motion, half-page scrolling, and a count prefix, and
Settings → Keys documents them so the new keys are discoverable in the app.

## What

- `e` moves to the last cell of a word. Copy mode's word family is
  whitespace-delimited with no `iskeyword` notion, so `w` behaves like vim's `W`
  and `e` like vim's `E`. `E` itself stays unbound, matching how `W` already is.
- `Ctrl+D` and `Ctrl+U` move half a page. Guarded on the ctrl chord, so a bare
  `d` or `u` stays unbound and AltGr keeps working on Windows. Reuses the
  existing `keys::is_ctrl_chord` rather than adding a rule.
- A typed count repeats the next motion (`12j`, `3w`), and a count with `g` or
  `G` jumps to that retained row the way vim's `NG` does. Rows here are physical
  terminal rows, so a wrapped line counts as more than one.
- The pending count shows in the status bar until a motion consumes it,
  mirroring vim's showcmd. Without it a typed `12` looks like a dead keypress.
- `0` keeps its existing meaning of "first column" unless a count is already
  being typed, so no current binding changes behaviour.
- An unrecognised key clears a pending count and is still swallowed rather than
  forwarded. Copy mode never lets a stray key reach the program in the pane.
- Counts are capped at four digits and word motions break as soon as they stop
  moving, so a large count cannot hold the engine lock the PTY reader needs.
- The status row still lists only movement, anchor, copy and cancel. The new
  motions live in Settings → Keys instead, because the row is clipped rather
  than wrapped and anything added to it pushes `q cancel` off an 80-column
  terminal. A new test pins that at the width the other bar tests use.
- Copy mode's status label and hints now come from the catalog. It was the only
  mode still holding hardcoded English while SCROLL and RESIZE were translated.
- Settings → Keys lists the new motions. Its copy-mode block read `w / B`, which
  this change would otherwise have made wrong.
- `every_language_covers_the_complete_settings_reference` now also asserts, per
  language, that each reference section has one description per key row. The
  Keys tab zips the two, so a mismatch silently dropped rows in one language
  only.

## Why

Copy mode's motions stopped at `w` and `B` with no counts, so selecting more
than a few words meant repeating keys.

`w` also cannot select a single word on its own. The selection is inclusive of
the cursor cell, so landing on the next word's first cell always takes one cell
and a space too many. `e` is what makes "select this word, copy it" exact, which
is the most common thing copy mode is used for.

Copy-mode keys are not rebindable, so Settings → Keys and the status bar are the
only places inside the app that can teach them. Adding motions without updating
those would have left the app describing a key set that no longer matched.

## Verification

- [x] `cargo test --locked`, 945 passed
- [x] `cargo clippy --all-targets -- -D warnings`
- [x] `cargo fmt --all --check`
- [x] Manual testing

Manual: ran the debug client and server from a plain terminal outside my own
Luvus session, on a temporary `LUVUS_HOME`. Confirmed the count badge appears
and clears, that the status row still ends in `q cancel` at 80 columns, and that
with `"language": "id"` the row renders in Indonesian. Also stepped through
Settings → Keys down to the copy-mode block and read every row.

## Screenshots

Settings → Keys, copy mode block. The `g / G` row is new, and `^D / ^U` carries
the `(^ is Ctrl)` gloss.

Settings Keys copy mode block

Copy mode with a pending count. The badge reports the typed count the way vim's
showcmd does, and `q cancel` keeps its place.

Copy mode status row with a pending count

Keyboard selection spanning retained rows.

Keyboard selection across retained rows

## Notes

- No visual block mode and no paragraph motions. The selection stays row-major,
  as before.
- Tested on macOS only. The one platform-sensitive path is the ctrl-chord check,
  which reuses an existing helper.

## Summary by CodeRabbit

- **New Features**
  - Added Vim-style counted motions in copy mode, including word-end jumps, half-page navigation, repeated motions, and direct row jumps.
  - Added pending-count feedback and expanded copy-mode keyboard guidance.
  - Added horizontal mouse scrolling and mode-aware diff scrolling.
  - Added persistent sidebar FILES placement and Korean localization.
  - Improved command inspection by detecting the process root when needed.

- **Documentation**
  - Expanded scrollback documentation with copy-mode motions, counts, navigation, selection, and cancellation behavior.

- **Bug Fixes**
  - Improved boundary handling and kept status guidance readable on narrow screens.

### CODE REVIEW

**Comment by RizRiyz** [comment]
@yosephbernandus welcopme to luvus! you need to fix these:

- Keep `q cancel` visible at 80 columns for every supported language, including count `9999`.
- Document `N` + `g`/`G` in Settings → Keys for every language.
- Add focused tests covering the maximum count across all language catalogs.

File: src/app/input.rs [hunk 1]
```diff
@@ -33,6 +33,49 @@ fn copy_word_forward(
...
+/// Row-local on purpose: a retained row is a physical terminal row, and `w`
+/// already treats a row edge as a break, so the trailing scan never crosses one.
+fn copy_word_end(
+    row_count: usize,
+    mut row_layout: impl FnMut(usize) -> Option<crate::terminal::vt::RetainedRowLayout>,
+    at: (usize, usize),
+) -> (usize, usize) {
+    // Step off the current cell first, so pressing `e` again advances instead of
+    // parking on the same word end.
+    let mut cur = (at.0, at.1.saturating_add(1));
```
[the remark this answers is not in the document]
Comment (yosephbernandus) [reply src/app/input.rs:79]: Not a bug. `e` on a one-cell word moves to the end of the next word because the cursor already sits on the current word's end. Vim behaves the same way on `a bb ccc` from column 0: `e` lands on column 3, a second `e` on column 7. Copy mode lands on those same two columns.

The fixture gap is real though. The existing test used `你好 world again`, where no word is a single cell, so that case was untested rather than broken. Added assertions for it in `copy_mode_word_end_stops_on_the_last_cell_of_each_word`.

File: src/ui/status.rs [hunk 2]
```diff
@@ -101,13 +101,18 @@ fn fixed_guidance(app: &App, t: &Theme) -> (Line<'static>, bool) {
...
         left.push(Span::raw("  "));
-        left.extend(hint("hjkl arrows", "move", t));
-        left.extend(hint("v", "anchor", t));
-        left.extend(hint("y", "copy", t));
-        left.extend(hint("q", "cancel", t));
+        // Vim's showcmd: a typed count is invisible otherwise, so `12j` looks
+        // like a dead keypress until the motion lands.
+        if copy.pending_count > 0 {
+            left.extend(hint(&copy.pending_count.to_string(), cat.copy_count, t));
+        }
```
[the remark this answers is not in the document]
Comment (yosephbernandus) [reply src/ui/status.rs]: Correct, and worse than the estimate. Measured against the 72 columns the row actually gets after the version chip, six of eight catalogs lose `q cancel` at a two-digit count:

```
count=0     de 72                                     (zero margin)
count=12    es 77  pt 80  fr 78  de 84  id 75  ja 79
count=9999  es 79  pt 82  fr 80  de 86  id 77  ja 81
```

English passed only by being the shortest at 70. German was already at exactly 72 before the count badge, so shrinking the badge could not have fixed it on its own.

Optional hints now give way in order instead of the row clipping, the anchor first and then the arrows. Copy and cancel never drop. The test walks all eight catalogs at the maximum count, and also asserts that an uncrowded row still carries everything, so a bug that always drops cannot pass it.

File: src/ui/status.rs [hunk 3]
```diff
@@ -101,13 +101,18 @@ fn fixed_guidance(app: &App, t: &Theme) -> (Line<'static>, bool) {
...
+        left.extend(hint("hjkl arrows", cat.act_move, t));
+        left.extend(hint("v", cat.copy_anchor, t));
+        left.extend(hint("y", cat.act_copy, t));
+        left.extend(hint("q", cat.act_cancel, t));
```
[the remark this answers is not in the document]
Comment (yosephbernandus) [reply src/ui/status.rs]: Done, and the four-digit case was not the only one failing. Against the 72 columns left of the version indicator, six of eight catalogs lose `q cancel` at a two-digit count, and German sat at exactly 72 before the count badge existed. Reserving width by shortening the badge could not have worked for that reason.

Optional hints now give way in order, the anchor first and then the arrows, so copy and cancel keep their place. The test loops every catalog at `COPY_COUNT_MAX` as you asked, and adds one assertion in the other direction: an uncrowded row has to keep all its hints, so a bug that always drops cannot pass.

File: src/i18n/settings.rs [hunk 4]
```diff
@@ -4,10 +4,28 @@
...
 pub const KEY_REFERENCE_KEYS: &[&[&str]] = &[
     &["h j k l", "q", "X", "-", "⇥ / ⇧⇥", "prefix ×2"],
     &["Shift+↑", "j / k", "Space / b", "g / G", "1–9", "q  esc"],
-    &["arrows  hjkl", "w / B", "Space / b", "v", "y  ⏎", "q  esc"],
+    &[
+        "arrows  hjkl",
+        "w / e / B",
+        "Space / b",
+        "^D / ^U",
+        "1–9",
```
[the remark this answers is not in the document]
Comment (yosephbernandus) [reply src/i18n/settings.rs:25]: Correct, and it points at something older. Copy mode's block had no `g / G` row at all while scroll mode has had one, so those keys worked and the in-app reference never listed them.

Added a `g / G` row in the position scroll mode uses, described as "oldest / newest row, so 5G jumps to row 5" across all eight catalogs, which covers the plain and the counted form together.

Rendering that block to check the new row turned up a clip in it too. The Indonesian count description was 63 characters and truncated mid-word in the panel. Shortened, and there is now a test that renders the copy-mode block in every language and asserts each description appears in full.

**Comment by yosephbernandus** [comment]
Thanks! @RizRiyz dude just moved my daily workflow from herdr to this seems like comfortable to me and feels good!

Also All three are in fc38d25, which I pushed about forty minutes after this comment, so the two likely crossed.

**`q cancel` at 80 columns in every language, including 9999.** The row gets 72 columns left of the version indicator, and the problem was wider than the review reported. Six of eight catalogs lost cancel at a two-digit count, and German sat at exactly 72 with no count at all:

```
count=0     de 72                                     (zero margin)
count=12    es 77  pt 80  fr 78  de 84  id 75  ja 79
count=9999  es 79  pt 82  fr 80  de 86  id 77  ja 81
```

German having no margin before the count badge existed means shrinking the badge could not have fixed it. The optional hints now give way in order instead, the anchor first and then the arrows, and copy and cancel never drop.

**`N` + `g`/`G` in Settings → Keys.** Copy mode's block had no `g / G` row at all while scroll mode has had one, so the keys worked and the reference never listed them. Added in the position scroll mode uses, described as "oldest / newest row, so 5G jumps to row 5" in all eight catalogs, which covers the plain and the counted form together.

**Focused tests at the maximum count.** Two, both looping `i18n::LANGS` on an 80-column terminal:

- `copy_mode_guidance_keeps_copy_and_cancel_in_every_language` renders at `COPY_COUNT_MAX` and asserts the row fits the budget it actually gets, read from `version_rect` rather than restated, and that copy and cancel are both present. A second case asserts an uncrowded row keeps every hint, so a bug that always drops cannot pass.
- `copy_mode_reference_rows_are_not_clipped_in_any_language` renders the Keys panel and asserts every copy-mode description appears in full.

I reverted each fix and confirmed the matching test fails before trusting either.

The second test caught something outside the review: the Indonesian count description was 63 characters and truncated mid-word in the panel. Shortened to 58.

**Comment by RizRiyz** [comment]
Approved @yosephbernandus thanks for making luvus better
